packages/features/ee/common/components/v2/LicenseRequired.tsx

```diff
@@ -11,21 +11,24 @@ type LicenseRequiredProps = {
   className?: string;
   role?: AriaRole | undefined;
   children: React.ReactNode;
+  toHide?: boolean; // if true, the component will be hidden instead of showing the empty screen
 };
 
 /**
  * This component will only render it's children if the installation has a valid
  * license.
+ * @param toHide - will hide the component with a license instead of showing the empty screen
  */
-const LicenseRequired = ({ children, as = "", ...rest }: LicenseRequiredProps) => {
+const LicenseRequired = ({ children, as = "", toHide = false, ...rest }: LicenseRequiredProps) => {
   const session = useSession();
   const { t } = useLocale();
   const Component = as || Fragment;
   const hasValidLicense = session.data ? session.data.hasValidLicense : null;
+
   return (
     <Component {...rest}>
       {hasValidLicense === null || hasValidLicense ? (
-        children
+        !toHide && children
```

Comment:
@zomars does this logic check out?

I want to:

hide the children if a license is valid